.pylintrc

```diff
@@ -0,0 +1,407 @@
+[MASTER]
```

Comment:
I generated it with `--generate-rcfile`, one of the things I had to modify for instance was:

```diff
-method-rgx=[a-z_][a-z0-9_]{2,30}$
+method-rgx=[a-z_][a-z0-9_]{2,50}$
```

Would be hard to do that without a fully generated config, my guess is they generate it like that because there are many options like this. I agree it's not ideal though, let me try to simplify it a bit.